lab1.asm

```diff
@@ -0,0 +1,20 @@
+
+; You may customize this and other start-up templates; 
+; The location of this template is c:\emu8086\inc\0_com_template.txt
+
+org 100h
+
+x dw ?
+y dw ?
+m dw 2
+c dw 5
+ 
+
+start:
+    printn 'Enter a value of x'
+   
```

Comment:
Print a new line here after the `Enter value` prompt. Otherwise this looks good :100: 
```suggestion
   putc 10
   putc 13
```

   
```
```